### DISCUSSION TITLE
AWS Authentication with MinIO

### DISCUSSION THREAD

**Discussion by canob** [grafana/grafana-infinity-datasource#688] [Q&A] [answered]
Hi everyone,

Does anybody know if it is possible to use AWS Authentication type of Infinity Plugin with MinIO?
The idea is to do S3 Select, https://docs.aws.amazon.com/AmazonS3/latest/API/API_SelectObjectContent.html, using MinIO API, with is compatible with AWS S3 API.

Thanks in advance for your help.

**Comment by samerbahri98** [comment, marked as answer]
I randomly found this while trying to do the same thing.
the answer is yes. you can query json files that you put in minio and use them for grafana dashboard.
you need to set the following:

- Region: your minio region (default is us-east-1)
- Service: s3
- Access key and secret key to the ones that you set up
- Allowed hosts: your minio host (example: http://minio:9000 )

nothing else needs to be set up. you can click on save&test
